source/partial-deep.d.ts

```diff
@@ -33,8 +33,8 @@ settings = applySavedSettings({textEditor: {fontWeight: 500}});
 @category Set
 @category Map
 */
-export type PartialDeep<T> = T extends Primitive
-	? Partial<T>
```

Comment:
Intentional that you're removing the `Partial`?

Reply:
Yes. Couldn't really get `RegExp` and `Date` to work with it there. Are there any drawbacks to removing it? I'll admit I don't fully understand the advantages of doing `Partial` with primitives and `RegExp`/`Date`.

Reply:
> Are there any drawbacks to removing it?

I don't know. I recommend using Git blame to figure out who added it and then mentioning them here.

Reply:
@kainiedziela I see you made the most recent change to this, which was actually adding this 2 years ago. Any concerns?

Reply:
No concerns